### PR TITLE
Adding docs for how to submit results

### PR DESCRIPTION
**Pull request by hannahbaumann**
## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: docs/source/private/overview.rst [hunk 1]
```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
...
 3. A CSV file with experimental data that you will need to prepare.
 
+To prepare the results for the submission, please follow these steps:
+
+1. Run the :ref:`post-simulation cleanup` script.
+2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
+3. Prepare a CSV file containing the experimental data for the set (see :ref:`here <exp-csv-file>`).
+4. Fill out the :ref:`private dataset info form <private info form>`.
+5. Create a folder named ``dataset_name`` where `dataset_name` can be any arbitrary name that you want to give the different private datasets
+6. Copy the three items, the compressed folder ``results_data.zip``, the experimental CSV file ``experimental_data.csv`` and the private info form ``info_form_private_sets.txt`` into the ``dataset_name`` folder.
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:275]:
```suggestion
6. Copy the three items, the compressed folder ``results_data.zip``, the experimental CSV file ``experimental_data.csv``, and the private info form ``info_form_private_sets.txt`` into the ``dataset_name`` folder.
```

File: docs/source/private/overview.rst [hunk 2]
```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
 You can open and edit the file using e.g. VIM or Microsoft Word. Please fill out this form for each private dataset and upload
 the file (as a pdf or text file) together with the other results in your zenodo upload.
 
+.. _private-data-gather:
 
 How will this be gathered?
```
Comment (jthorton) [inline docs/source/private/overview.rst:259]: Lets make this clear and call it `Preparing results upload` or something similar.

File: docs/source/private/overview.rst [hunk 3]
```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
...
 ==========================
 
-.. _private-data-gather:
-
 We anticipating gathering the data through:
 
 1. A script provided by the OpenFE team that will extract relevant information from your simulations.
 2. A text file for you to fill in with information.
 3. A CSV file with experimental data that you will need to prepare.
```
Comment (jthorton) [inline docs/source/private/overview.rst:266]: No longer needed.

File: docs/source/private/overview.rst [hunk 4]
```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
...
 
+To prepare the results for the submission, please follow these steps:
+
+1. Run the :ref:`post-simulation cleanup` script.
+2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
```
Comment (jthorton) [inline docs/source/private/overview.rst:271]: Maybe add that the contents are all human-readable and should be reviewed before sharing?

**Review by jthorton — CHANGES_REQUESTED** [review]
It looks good. I would like the result gathering heading on the sidebar to be clearer, and the other changes should be considered optional.

File: docs/source/private/overview.rst (hunk 4 above)
Comment (hannahbaumann) [reply docs/source/private/overview.rst:265]: Added this!

File: docs/source/private/overview.rst (hunk 3 above)
Comment (hannahbaumann) [reply docs/source/private/overview.rst]: Removed this.

File: docs/source/private/overview.rst (hunk 2 above)
Comment (hannahbaumann) [reply docs/source/private/overview.rst]: Changed this.

File: docs/source/private/overview.rst [hunk 5]
```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
 You can open and edit the file using e.g. VIM or Microsoft Word. Please fill out this form for each private dataset and upload
 the file (as a pdf or text file) together with the other results in your zenodo upload.
 
+.. _private-data-gather:
 
-How will this be gathered?
-==========================
+Upload of results
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:259]:
```suggestion
Uploading the results to zenodo
```

File: docs/source/private/overview.rst [hunk 6]
```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
 You can open and edit the file using e.g. VIM or Microsoft Word. Please fill out this form for each private dataset and upload
 the file (as a pdf or text file) together with the other results in your zenodo upload.
 
+.. _private-data-gather:
 
-How will this be gathered?
-==========================
+Uploading the results to zenodo
+=================
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:260]:
```suggestion
Uploading the results to zenodo
========================
```

File: docs/source/private/overview.rst [hunk 7]
```diff
@@ -250,17 +254,26 @@ We provide a template text file for you to fill in all the information. You can
...
-.. _private-data-gather:
+To prepare the results for the submission, please follow these steps:
 
-We anticipating gathering the data through:
+1. Run the :ref:`post-simulation cleanup` script.
+2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
+   The contents in this folder are all human readable and should be reviewed before submitting.
+3. Prepare a CSV file containing the experimental data for the set (see :ref:`here <exp-csv-file>`).
+4. Fill out the :ref:`private dataset info form <private info form>`.
+5. Create a folder named ``dataset_name`` where `dataset_name` can be any arbitrary name that you want to give the different private datasets
```
Comment (jthorton) [inline docs/source/private/overview.rst:269]: Is the second dataset_name supposed to be highlighted it might be missing the second set of `

Comment (IAlibay) [reply docs/source/private/overview.rst:269]:
```suggestion
5. Create a folder named ``dataset_name`` where ``dataset_name`` can be any arbitrary name that you want to give the different private datasets
```
Just to make it easier to merge in.

File: docs/source/public/overview.rst [hunk 8]
```diff
@@ -721,6 +720,33 @@ The results directory should be organised as follows:
...
+
+The results from this benchmarking study will be submitted to the OpenFE team via a zenodo upload.
+
+To upload the data, please follow these steps:
+
+1. Run the :ref:`post-simulation cleanup` script.
+2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
+3. Upload the compressed results folder to zenodo.
+
+  a) Go to `zenodo <https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest>`_ to upload the data to the Open Free Energy organization.
```
Comment (IAlibay) [inline docs/source/public/overview.rst:736]:
```suggestion
  a) Go to `zenodo <https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest>`_ to upload the data to the **Open Free Energy Zenodo Community**.
```
Bolding it to increase attention and also slightly pendantic "community"

**Review by IAlibay — APPROVED** [review]
lgtm thanks!

**Comment by hannahbaumann** [comment]
@IAlibay @jthorton Should we add something about confirming the OpenFE version used in the zenodo upload?

**Comment by jthorton** [comment]
@hannahbaumann yes good idea maybe we could add the section mike suggested like did you make any changes to the environment or we could ask them to upload the exported environment yaml file?

**Review by jthorton — APPROVED** [review]
Great job!

**Pull request merged**